libs/torii_utils/query_client.hpp

```diff
@@ -48,6 +48,8 @@ namespace torii_utils {
     std::unique_ptr<iroha::protocol::QueryService::Stub> stub_;
     grpc::CompletionQueue completionQueue_;
     grpc::Status status_;
+
+    enum class State { RequestCreated, ResponseSent };
```

Comment:
Add description for the enum.

Reply:
Please check my answer to @x3medima17 (very first issue).